raiden-dapp/src/services/raiden-service.ts

```diff
@@ -52,10 +52,20 @@ export default class RaidenService {
         this.store.commit('account', await this.getAccount());
         this.store.commit('balance', await this.getBalance());
 
-        this.subscription = raiden.channels$.subscribe(value => {
-          this.store.commit('updateChannels', value);
+        this.subscription = raiden.events$.subscribe(value => {
+          if (value.type === RaidenActionType.SHUTDOWN) {
+            this.store.commit('reset');
+            raiden.stop();
```

Comment:
No need to `stop` after `RaidenActionType.SHUTDOWN`, as the former is also a shutdown event, and everything that happens after shutdown is ignored

Reply:
That is good to know thank you :)